extruct/pathfinder.py

```diff
@@ -0,0 +1,62 @@
+import itertools
```

Comment:
Is `pathfinder.py` module really needed? find function is imported in opengraph.py, but it is unused. I'd prefer to remove it.